modules/cis-1-2-rules/main.tf

```diff
@@ -0,0 +1,39 @@
+locals {
+  all_rules       = module.aws_config_rules_yaml_config.map_configs
+  rules_with_tags = { for k, rule in local.all_rules : k => rule if rule.tags != null }
+
+  cis_1_2_tag                   = "compliance/standards/aws-foundations-cis/1.2"
+  cis_1_2_tagged_rules          = { for key, rule in local.rules_with_tags : key => rule if lookup(rule.tags, local.cis_1_2_tag, false) == true }
+  cis_1_2_logging_account_rules = { for key, rule in local.cis_1_2_tagged_rules : key => rule if var.is_logging_account && lookup(rule, "loggingAccountOnly", false) }
+  cis_1_2_global_resource_rules = { for key, rule in local.cis_1_2_tagged_rules : key => rule if var.is_global_reource_region && lookup(rule, "globalResource", false) }
+  cis_1_2_base_rules            = { for key, rule in local.cis_1_2_tagged_rules : key => rule if(! lookup(rule, "loggingAccountOnly", false) && ! lookup(rule, "globalResource", false)) }
+  cis_1_2_all_rules             = merge(local.cis_1_2_base_rules, local.cis_1_2_logging_account_rules, local.cis_1_2_global_resource_rules)
+  cis_1_2_enabled_rules = { for key, rule in local.cis_1_2_all_rules : key => {
```

Comment:
Since now this submodule is scoped to `cis-1-2`, we can drop the `cis_1_2_` prefixes as it's no longer ambiguous.

Reply:
38d724e7a8754b328a4e66d548dfa4cd6c597ab0